lib/boxes.js

```diff
@@ -23,7 +23,18 @@ async function setBoxByTitle(title, item) {
   return postBox;
 }
 
+async function setItemByTitle(title, item) {
+  const addItem = await collection("boxes").updateOne(
+    {
+      title: title,
```

Comment:
Code style (up to you)
```suggestion
      title,
```